students/andykwon/lesson07/html_render.py

```diff
@@ -77,17 +92,64 @@ class Title(OneLineTag):
 
 class SelfClosingTag(Element):
 
-    def render(self, outfile)
-        pass
+    def append(self, *args, **kwargs):
+        raise TypeError
```

Comment:
Good to add a message to the error:

`raise TypeError("you cannot add content to a Self Closing Tag")`